Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -426,6 +428,27 @@ def fromString(value):
 	def toString(value):
 		return fl2str(value, 14)
 
+class Angle(F2Dot14):
```

Comment:
This `Angle` class is in degrees; that choice is not reflected in the name. I don't have a good suggestion though.

Reply:
what about `DegreeAngle`?

Reply:
Either that or AngleDegrees. Both are ugly. Maybe just Angle is fine.

Reply:
Perhaps the name `Angle` with a comment indicating it's degrees and describing how it maps on read/write with a link to learn more?

Reply:
I kept the name `Angle` and added a comment in 60f072414478dba95c743e8f6bbdb5792088c71b